### PR TITLE
Category project association; closes #162

### LINKED ISSUE #162: If an admin can create a category, there needs to be a project associated with it otherwise the categories controller will break in the show.

### PR DESCRIPTION
**Pull request by AdamSmith910**
I cleaned up the style of the admin/tenants/index page so it looked better by switching the <th> alignment from center to left

I removed the redundant Tenants link from the admin dashboard because the borrowers link is already in the admins navbar

I created a message that displays for the categories/show if the category in question does not have an projects associated with it

I wrote tests for the above

### CODE REVIEW

**Comment by oorion** [comment]
I don't see any tests here.  Also, I think we should hold off until Mike's PR is merged.  What do you think?

File: app/views/categories/show.html.erb [hunk 1]
```diff
@@ -2,19 +2,25 @@
...
-        <%= f.submit 'Lend', class: "btn btn-default" %>
       <% end %>
-    </div>
+    <% else %>
+      <div class="box col-lg-3">
+        There are no projects for this category
+      </div>
     <% end %>
   </div>
 </div>
```
Comment (EmilyMB) [inline app/views/categories/show.html.erb]: I think we can delete this whole file, right? None of our user stories ask us have a show page for each category.

File: test/integration/category_index_test.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    category = create(:category)
+
+    visit root_path
+
+    refute page.has_content?(category.name)
+  end
+
+  test "Categories index page does display a category that
+  has an associated project" do
+    # the factory for project automatically creates an associated
```
Comment (EmilyMB) [inline test/integration/category_index_test.rb]: Maybe remove these comments?

**Comment by AdamSmith910** [comment]
No longer relevant after ajax

**Pull request closed without merging**